src/api/app/views/webui2/shared/_repositories_flag_table_column.html.haml

```diff
@@ -1,19 +1,21 @@
 :ruby
-  flag = effective_flag(flags, repository, architecture)
-  is_flag_set_by_user = flag_set_by_user?(flags, repository, architecture)
+  flag = flags.effective_flag(repository, architecture)
+  is_flag_set_by_user = flags.set_by_user?(repository, architecture)
   title = flag.status.capitalize
-  title += is_flag_set_by_user ? ' (set by user)' : ' (calculated)'
-  if user_can_set_flags
-    content = render(partial: 'shared/flag_popover',
-                     locals: { flag: flag, flags: flags, repository: repository, architecture: architecture,
-                               project: project, package: package })
-  end
   icon_class = flag.status == 'disable' ? 'fa-ban text-danger' : 'fa-check text-success'
-  icon_class += ' ml-0_6' if is_flag_set_by_user
+  data = { status: flag.status, flag: flag.flag, repository: repository,
```

Comment:
next thing to do: this is really a helper function now - most is ruby (that's the reason I left the helper module empty and didn't remove it)